### DISCUSSION TITLE
Whether there is a lack of consideration for multiple planes

### DISCUSSION THREAD

**Discussion by bianyu91** [MoatLab/FEMU#135] [Q&A]
When I read the following code snippet of do_gc() function in bbssd/ftl.c， I‘m wondering why the multi-plane scenario is not taken into count.

` 
/* copy back valid data */
    for (ch = 0; ch < spp->nchs; ch++) {
        for (lun = 0; lun < spp->luns_per_ch; lun++){

            ppa.g.ch = ch;
            ppa.g.lun = lun;
            ppa.g.pl = 0;
....
}
`

**Comment by huaicheng** [comment]
You're correct, we simply assume 1 plane per die, which is a simplified and outdated NAND model. Happy to take your patch for improvement if you intend to work on it.
